packages/paste-website/src/component-examples/ComboboxPrimitiveExamples.ts

```diff
@@ -23,8 +23,7 @@ const BasicCombobox = () => {
         <Input
           id={uid}
           type="text"
-          {...getInputProps()}
-          {...getToggleButtonProps({tabIndex: 0})}
+          {...getInputProps(getToggleButtonProps({tabIndex: 0}))}
```

Comment:
Unfortunately the downshift docs are not complete and [there is content in the `useMultipleSelection` documentation about strategies to combine these prop getters](https://github.com/downshift-js/downshift/tree/master/src/hooks/useMultipleSelection#prop-getters) that is missing in `useCombobox`.

The toggle button prop getter doesn't actually return button-specific attributes, and since we are using the input box as the menu/dropdown trigger, this is as intended. It's actually the same thing we do in `Combobox`, with the extra getter for the multiple selection hook.

Reply:
The way I'm reading that, from what I understand it is saying to do:

```js
getDropdownProps(getToggleButtonProps({})
or
getDropdownProps(getInputProps({})

but not

getInputProps(getToggleButtonProps({})
```

Reply:
If we revert this change, does anything break?